workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Editor/Prefabs/PrefabPreprocessor.cs

```diff
@@ -9,6 +9,13 @@
 
 namespace Improbable.Gdk.GameObjectRepresentation.Editor
 {
+    /// <summary>
+    ///     A prefab processor which disables any Monobehaviours on GameObjects which have fields with the
```

Comment:
but only on the parent gameobject and only if the prefab is in the `Resources` folder